cmd/minikube/cmd/root.go

```diff
@@ -290,3 +297,8 @@ func getClusterBootstrapper(api libmachine.API, bootstrapperName string) (bootst
 
 	return b, nil
 }
+
+func addToPath(dir string) {
+	path := os.Getenv("PATH")
```

Comment:
does this work on windows as well ?

Reply:
@blueelvis do you know if this is okay for windows ?

Reply:
I don't think it works for windows. That's why the caller validates the OS before calling it. It was actually validated inside the function before, but after a review from Anders, he asked to change it.